Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -1743,6 +1743,9 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
 
                 if cbc.scope.evaluate(BuiltinMacros.SWIFT_ENABLE_INCREMENTAL_COMPILATION) {
                     args.append("-incremental")
+                    if !cbc.scope.evaluate(BuiltinMacros.SWIFT_DISABLE_INCREMENTAL_SCAN) {
```

Comment:
Can we add a check here that the driver supports the flag before we use it? `LibSwiftDriver.supportsDriverFlag(spelled: "-incremental-dependency-scan")`

Reply:
Done!

Reply:
Ah, looks like there are a lot of tests which verify the command-line sequence that get tripped up with a new flag.